client/src/Pages/Store.js

```diff
@@ -70,27 +102,27 @@ const Store = () => {
     // Store/item data goes to <ItemGrid items = {}
     return (
         <div className="store">
-            <Menu 
-                selection = {storeStatus.selection}
-                updateSelection = {updateSelection}
-            />
-            <ItemGrid
-                items = {itemData}
-                selection = {storeStatus.selection}
-                itemOnClick = {itemOnClick}
+            <Menu
+                selection={storeStatus.selection}
+                updateSelection={updateSelection}
             />
+            {itemData && <ItemGrid
```

Comment:
This is the only part that changes the behaviour of the return.
I auto formatted the document and it removed all the space next to the`=`s 😅